### DISCUSSION TITLE
stream 只能收发4GB数据

### DISCUSSION THREAD

**Discussion by 359547629** [alibaba/xquic#297] [General]
stream收发4GB数据后，收不到任何事件通知，也无close的事件通知； 修改xqc_trans_settings_t里面的max_stream_data_uni，max_data等参数也无任何效果

**Comment by ruiqizhou** [comment]
Hi, Could you please submit a [Github Issue](https://github.com/alibaba/xquic/issues/new?assignees=&labels=&template=bug_report.yml&title=%5BBug%5D%3A+) that includes as much detail as possible(which version, what environment, etc) and **includes steps to reproduce the problem**? We will check ASAP.

**Comment by MPK1** [comment]
My chinese is not that good, but I think I experience a similar error. When sending files larger than exactly 2 GiB, the transmission fails and the connection is not closed properly. I'm currently trying to fix it, if I either get it working or not, I'll submit an issue. If anyone has some ideas on buffers that could cause this issue, please tell me.
